src/pages/routes.tsx

```diff
@@ -1,17 +1,24 @@
 import { createBrowserRouter } from "react-router-dom";
+import Layout from "./Layout.tsx";
+import EmployeeListing from "./EmployeeListing/EmployeeListing.tsx";
+import ErrorPage from "../components/ErrorPage/ErrorPage.tsx";
 
-
-const router = createBrowserRouter(
+const router = createBrowserRouter
+(
   [
     {
-      element: <></>,
+      element: <Layout></Layout>,
       children: [
         {
           path: "/",
-          element: <></>,
+          element: <EmployeeListing />,
+        },    
+        {
+          path: "/error?statusCode=:statusCode",
+          element: <ErrorPage />,
         },
       ],
-      errorElement: <></>,
+      errorElement: <ErrorPage />,
     },
   ],
   { basename: import.meta.env.DEV ? "/" : "/HRM-App-React/" }
```

Comment:
Change repo name.

Reply:
Resolved in c1d862b1e940779d808ab55886138a9ced297f02